apps/admin_app/lib/admin_app_web/templates/order/show.html.eex

```diff
@@ -125,7 +125,11 @@
     <div class="col-8">
       <%= render "line_item.html", %{order: @order} %>
       <div class="card-body">
-        <%= render_invoice_links(@order) %>
+        <div>
+        <a class="btn btn-primary" href="/orders/qEsmxy4f89Ur5LH0rEmm6/download-invoice" style="margin-right: 3px;">Download Invoice</a>
```

Comment:
You can consider using Phoenix `link`  function here. Please remove the hardcoded order number here.